### ISSUE TITLE
QR Code <img> missing alt text

### ISSUE THREAD

**Issue by jjjjjjjj2018** [davidshimjs/qrcodejs#257]

**Comment by tarcisioreis** [comment]
In the code, the text is dynamically generated in the div and not the image, see the printscreen

![image](https://user-images.githubusercontent.com/7525518/140407072-8f41bdb0-02f6-4f85-8555-90f072845ea6.png)

Hope this helps.
